### PR TITLE
Feat: My cart

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created check page 

Screenshot (204)

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks good to me

File: CSS/cart.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+@import url(../CSS/main.css);
+
+.cart-container {
+  min-height: 85vh;
+}
+.images-card {
+  border: 0.5px solid var(--border-color);
+  border-radius: 0;
+  box-shadow: none;
+  background-color: #fff;
```
Comment (srishti-maurya) [inline CSS/cart.css:11]: maybe use may use color variable here

File: CSS/cart.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  min-height: 320px;
+  overflow: hidden;
+  width: 100% !important;
```
Comment (srishti-maurya) [inline CSS/cart.css:14]: Suggestion: you may avoid using `!important`

**Review by srishti-maurya — APPROVED** [review]
UI Looks amazing

**Comment by preethasri** [comment]
Looks good ,👍

**Comment by BelideNikhil** [comment]
I agree with Srishti.
Also try using webp/svg format images for faster loading.
Rest all looks good!

**Pull request merged**